src/xo-app/dashboard/health/index.js

```diff
@@ -20,29 +21,17 @@ import {
   noop
 } from 'utils'
 
-const AlarmMessage = connectStore(() => ({
+const TdObject = connectStore(() => ({
   object: createGetObject(
     (_, props) => props.message.$object
```

Comment:
If the name of the `prop` you pass to this component is `id` you don't need to use this id selector.